cicd/builds/advanced-build-operations.adoc

```diff
@@ -6,9 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The following sections provide instructions for advanced build operations including
-setting build resources and maximum duration, assigning builds to nodes, chaining
-builds, build pruning, and build run policies.
+The following sections provide instructions for advanced build operations including setting build resources and maximum duration, assigning builds to nodes, chaining builds, build pruning, and build run policies.
```

Comment:
I think "the following sections" might fall under the "avoid self-referential language" recommendation here https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc